bigtable-hbase-1.x-parent/bigtable-hbase-1.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestRpcRetryBehavior.java

```diff
@@ -210,6 +210,8 @@ protected Connection makeConnection(Map<String, String> customConnProps) throws
     for (Map.Entry<String, String> connProp : customConnProps.entrySet()) {
       config.set(connProp.getKey(), connProp.getValue());
     }
+    // retry on aborted to differentiate server hang an explicit server error
```

Comment:
It's been a while since I saw the code, but I want to confirm that this test is now using the veneer client. Wasn't sure if any other properties needed to be specified to enable the switch, or if the switch is done by default

Also, if we are switching to veneer, when is the old client-core being deleted :)

Reply:
The switch to veneer is happening in another PR, I extracted these changes from that PR

It'll be awhle until the old client gets deleted, we still have a lot of users